web/conftest.py

```diff
@@ -327,7 +327,7 @@ class Meta:
     effective_date = datetime(1900, 1, 1)
     publication_date = datetime(1901, 1, 1)
     updated_date = datetime(1902, 1, 1)
-    source_ref = factory.Sequence(lambda n: {"id": n})
+    source_ref = factory.Sequence(lambda n: n)
```

Comment:
This was just incorrect; the `source_ref` is a normal char field in the database:

```
 source_ref       | character varying(10000)   |           | not null | 
```